tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -53,6 +53,7 @@ COPY --from={{ app["name"] }}-src /openedx/app /openedx/app
 COPY --from={{ app["name"] }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
 ENV PUBLIC_PATH='/{{ app["name"] }}/'
 EXPOSE {{ app['port'] }}
+ENV APP_ID={{ app["name"] }}
 
 COPY ./env/production /openedx/env/production
 {%- set overrides = app.get("env", {}).get("production", {}) %}
```

Comment:
We should remove these (and the actual files in `env/`), the equivalent development lines, and also their corresponding `{%- if overrides %}` blocks below.